libs/core/tests/unit_tests/test_imports.py

```diff
@@ -1,14 +1,22 @@
 import glob
 import importlib
+import subprocess
 from pathlib import Path
 
+import pytest
+
 
 def test_importable_all() -> None:
     for path in glob.glob("../core/langchain_core/*"):
         relative_path = Path(path).parts[-1]
         if relative_path.endswith(".typed"):
             continue
         module_name = relative_path.split(".")[0]
+        result = subprocess.run(
```

Comment:
Perhaps split into another unit test? (Or maybe add a doc-string to explain what it's doing -- i realize I'm guilty of this as well, but as the test picks up complexity it'll be more difficult to remember why the code is what it is